SparkPost/2web2ui
# FAD-5253 fixes refresh token flow

To test:

* Checkout branch
* Log in with "keep me logged in" checked
* In "Application" panel, find auth cookie and change a few characters of the auth token
* Refresh any page, you should stay logged in
* Back in "Application" panel, find auth cookie and change the auth token _and_ the refresh token, refresh a page, it should log you out

Note: There were a few different issues, let me walk you through the PR before you jump in

tl;dr

* Log out didn't work because we were trying to `JSON.parse` the stored cookie twice, which was blowing up and causing the log out action to fail and be swallowed
* Refresh token didn't work because we often make multiple requests at once, so the first one was getting a new auth token but before it finished using it, the 2nd request also requested a new token using the same refresh token which was then invalid (they can only be used once), resulting in a 400 that caused the console errors.
